shared/js/background/chrome-events.es6.js

```diff
@@ -863,13 +872,11 @@ chrome.alarms.onAlarm.addListener(alarmEvent => {
         tdsStorage.getLists()
             .then(lists => trackers.setLists(lists))
             .catch(e => console.log(e))
+        configStorage.updateConfigData()
     } else if (alarmEvent.name === 'clearExpiredHTTPSServiceCache') {
         httpsService.clearExpiredCache()
     } else if (alarmEvent.name === 'updateUserAgentData') {
-        settings.ready()
-            .then(() => {
-                cookieConfig.updateCookieData()
-            }).catch(e => console.log(e))
+        // TODO: Reinstate user agent rotation
```

Comment:
Is this to be added after or in this PR?

Reply:
On second thought this isn't needed. If we bring the user agent stuff back we'll put it in the config which is already loaded elsewhere.